src/main/lua/exasolvs/request_dispatcher.lua

```diff
@@ -1,15 +1,42 @@
 local log = require("remotelog")
 local cjson = require("cjson")
-local adapter = require("exasolrls.adapter", "adapter")
+--local adapter = require("exasolrls.adapter", "adapter")
```

Comment:
If it's commented out, should it be removed?

Reply:
Artifact from before introducing `entry` object. Removed.